analyzers/python/python.go

```diff
@@ -163,15 +163,29 @@ func (a *Analyzer) Analyze() (graph.Deps, error) {
 	case "requirements":
```

Comment:
We should add an explicit `setuptools` case here that only runs `FromSetupPy`. Basically:
```go
case "setuptools":		
                 setupPyPath := filepath.Join(a.Module.Dir, "setup.py")
		reqs, setupErr := pip.FromSetupPy(setupPyPath)
		if setupErr == nil {
			return requirementsToDeps(reqs), nil
		}

		err.Troubleshooting = err.Troubleshooting + " OR " + setupErr.Troubleshooting

		return graph.Deps{}, err
```
It can be really helpful to mandate this. We may also want to modify the `requirements` case as well so that it no longer fallsthrough.

Reply:
should the combined requirements-fall-back-to-setup.py be used as a separate default?